chainer/computational_graph.py

```diff
@@ -190,10 +208,14 @@ def _skip_variable(nodes, edges):
 
 
 def build_computational_graph(
-        outputs, remove_split=True, variable_style=_var_style,
-        function_style=_func_style, rankdir='TB', remove_variable=False,
+        outputs, remove_split=True, variable_style=_default,
+        function_style=_default, rankdir='TB', remove_variable=False,
```

Comment:
Let's use the explicit value `'default'` here and make it documented.

Reply:
Then I'd like to have the doc like
```
        function_style(dict or 'default'): Dot node style for function.
            The ``'default'`` style is
            ``{'shape': 'record', 'fillcolor': '#6495ED', 'style': 'filled'}``.
```

I'm wondering if we can do it without repeating the default values.